data/eos-metrics-event-recorder.service.in

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=EndlessOS Metrics Event Recorder Server
-Wants=systemd-logind.service
```

Comment:
Jasper explained via IRC that, in practice, systemd starts the event recorder daemon concurrently with the login manager on account of this wants. That contributes to the user's perception of slow startup, and removing the wants, in practice, causes systemd to wait longer before starting the event recorder daemon. To be clear, there is nothing in the documentation of systemd that necessitates that systemd behave as such, but this is (we believe) the way it happens to be implemented.